src/models/InlineStyle.js

```diff
@@ -26,15 +26,20 @@ export default class InlineStyle {
     if (!generated[hash]) {
       const root = parse(flatCSS)
       const declPairs = []
+      const workaroundValues = {}
       root.each(node => {
         if (node.type === 'decl') {
-          declPairs.push([node.prop, node.value])
+          if (/border-?radius/i.test(node.prop)) {
```

Comment:
Let's add a comment here saying why we need this, linking to the relevant discussion.